diff -Naur irda-utils-0.9.18.orig/irdadump/Makefile irda-utils-0.9.18/irdadump/Makefile
--- irda-utils-0.9.18.orig/irdadump/Makefile	2006-07-11 08:16:06.000000000 +0200
+++ irda-utils-0.9.18/irdadump/Makefile	2007-06-14 17:55:23.000000000 +0200
@@ -5,6 +5,7 @@
 AR = ar
 RANLIB = ranlib
 RM = rm -f
+PKG_CONFIG = pkg-config
 
 RM_CMD = $(ECMD)$(RM) *.BAK *.bak *.o ,* *~ *.a
 
@@ -19,7 +20,7 @@
 SYS_LIBRARIES = 
 SYS_LIBPATH = -L/usr/lib -L/usr/local/lib 
 
-CFLAGS= -O2 -W -Wall -I. `pkg-config --cflags glib-2.0`
+CFLAGS= -O2 -W -Wall -I. `$(PKG_CONFIG) --cflags glib-2.0`
 
 LIBIRDADUMP_OBJS = irdadump.o irlap.o irlmp.o obex.o ircomm.o netbuf.o capture.o
 IRDADUMP_OBJS = main.o
@@ -40,7 +41,7 @@
 
 irdadump: $(IRDADUMP_OBJS) $(LIBIRDADUMP_TARGET)
 	$(prn_cc_o)
-	$(ECMD)$(CC) $(CFLAGS) `pkg-config --libs glib-2.0` -o  $(IRDADUMP_TARGET) $< $(LIBIRDADUMP_TARGET)
+	$(ECMD)$(CC) $(CFLAGS) `$(PKG_CONFIG) --libs glib-2.0` -o  $(IRDADUMP_TARGET) $< $(LIBIRDADUMP_TARGET)
 
 
 .c.o:
